### ISSUE TITLE
An urgent accessibility issue with dismissing banner by keyboard navigation

### ISSUE THREAD

**Issue by JayBox325** [a-digital/cookie-consent-banner#6]
I was just testing a site for keyboard-only users that navigate via tabs and because the dismiss button doesn't have an empty href property using 'return' doesn't dismiss it.

**Comment by mark-adigital** [comment]
Thanks - have added `#` as the href attribute now.

**Comment by JayBox325** [comment]
Hi @mark-adigital, long time no speak, this is a great solution and worked great, but my client has noticed that it jumps to the top of the page on click due to there being an empty # in the href. Are you able to update it with an empty href= attribute as: `<a href="">Got it</a>`.

This is valid HTML and it gives the opportunity for keyboard navigators to clear it, but it doesn't have the jump.

**Comment by mark-adigital** [comment]
Hi @JayBox325 - please see latest release (which includes a lot of other significant updates) for a fix to this. After some reading around, I settled on `javascript:void(0);` as the least of all evils.
